Examples/Layouts/MSDynamics_CX-6589_workaround.json

```diff
@@ -0,0 +1,90 @@
+{
+    "version": "0.0.2",
+    "appTitle": "Contact Center Desktop",
```

Comment:
can you please change the title to "Webex Contact Center", this is now being the default title inside default layout

Reply:
Added the changes.